packages/react/src/components/Stack/Stack.styles.ts

```diff
@@ -96,14 +103,47 @@ export const styles: IStackComponent['styles'] = (props, theme, tokens): IStackS
           // avoid unnecessary calc() calls if horizontal gap is 0
           width: columnGap.value === 0 ? '100%' : `calc(100% + ${columnGap.value}${columnGap.unit})`,
           maxWidth: '100vw',
+        },
+        !enableScopedSelectors && [
+          {
+            '> *': {
```

Comment:
Could this be rewritten to something like:

```ts
`> ${enableScopedSelectors ? GlobalClassNames.child : '*'}` ....
```

The way things are currently written introduces a fair bit of duplication which will regress bundle size in addition to maintenance burden.

Reply:
Yeah, that should work, I'll rewrite it that way.